cordova-lib/src/PlatformProject.js

```diff
@@ -257,11 +257,15 @@ function save() {
 PlatformProject.prototype.build = build;
 function build(opts) {
     var self = checkThis(this);
-    var bin = path.join(self.root, 'cordova', 'build');
-    var args = [];
 
-    var copts = { stdio: 'inherit' };
-    return superspawn.spawn(bin, args, copts);
+    return self.copyWww(opts.paths.www)
```

Comment:
We don't really want to copyWww as part of build. This disables a use case where the user outputs (e.g. from a gulp stream) directly to platform www dir. I agree, though that there should be a convenience function like updateWwwAndBuild because this is the most frequent use case.